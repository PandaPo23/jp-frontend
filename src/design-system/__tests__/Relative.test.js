import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Relative from '../Relative';

describe('Relative component', () => {
  it('should render as Relative', () => {
    const wrapper = mount(<Relative />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
