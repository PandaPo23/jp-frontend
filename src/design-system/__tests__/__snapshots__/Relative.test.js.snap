// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Relative component should render as Relative 1`] = `
.emotion-0 {
  position: relative;
  position: relative;
}

<Relative
  position="relative"
>
  <div
    className="emotion-0"
  />
</Relative>
`;
